nilearn/datasets/struct.py

```diff
@@ -869,7 +876,7 @@ def fetch_oasis_vbm(
 
 
 @fill_doc
-def fetch_surf_fsaverage(mesh="fsaverage5", data_dir=None):
+def fetch_surf_fsaverage(mesh="fsaverage5", data_dir=None, as_polymesh=False):
```

Comment:
pros:
- does not add a new function to the public API
cons:
- if set to True this only LOADS the meshes as PolyMesh where as when False this returns the paths to the meshes AND the data (so the name of the function is not quite right anymore)

Reply:
alternative: 
- do not touch fetch_surf_fsaverage
- have a new public function to load the fsaverage meshes
- have a new public function to load the fsaverage data